.github/workflows/codeql.yml

```
@@ -0,0 +1,50 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  schedule:
+    - cron: '24 1 * * 5'
```

Fix to make yamllint happy.
```suggestion
  - cron: '24 1 * * 5'
```